launch_testing_ament_cmake/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ if(BUILD_TESTING)
   if(NOT LAUNCH_TESTING_INSTALL_PREFIX)
       message(FATAL_ERROR "launch_testing package not found")
   endif()
+  file(TO_CMAKE_PATH "${LAUNCH_TESTING_INSTALL_PREFIX}" LAUNCH_TESTING_INSTALL_PREFIX)
```

Comment:
Would it make not sense to perform this step within the `add_launch_test` function called below?

Reply:
My thought was that converting the value immediately guarantees that it's a valid CMake path for any subsequent uses. I'm wasn't sure whether `file(TO_CMAKE_PATH ...)` would correctly handle a path with mixed path segments but a small emperical example suggests that it would.

I think both approaches have their advantages and I don't have an opinion on which advantages are preferred.

Reply:
I think it's better to place it inside `add_launch_test`. Then other packages using the function won't fall victim to the same issue. My two cents.